docker/jupyter/Dockerfile

```diff
@@ -15,6 +15,11 @@ RUN apk add --no-cache python3 py3-pip py3-pyzmq py3-grpcio
 # Install IPythono Notebook tutorials
 COPY /docker/jupyter/js /jupyter/js/
 COPY build/tutorial /workspace
+# Remove non-interactive tutorials
```

Comment:
Maybe we can add a TODO comment or an issue to recovery these tutorials?

Reply:
Yes, I will fire an issue.